docs/time.md

```diff
@@ -89,7 +89,7 @@ val app = Channel.unbounded[IO, String].toResource.flatMap { emailCh =>
     div(
       span(
         label("Your email: "),
-        input { self =>
+        input { (self: fs2.dom.HtmlInputElement[IO]) => // FIXME compiler bug
           onInput --> (_.evalMap(_ => self.value.get).through(emailCh.sendAll))
```

Comment:
This is extremely annoying. Another workaround is to do:
```scala
input { self =>
  Tuple1(onInput --> (_.evalMap(_ => self.value.get).through(emailCh.sendAll)))
}
```

I'm going to try and minimize and report a compiler bug.

Reply:
- https://github.com/lampepfl/dotty/issues/16737